vllm/compilation/backends.py

```diff
@@ -413,8 +413,9 @@ def __init__(
         # them, e.g. backbone (default), eagle_head, etc.
         self.prefix = prefix or model_tag
 
+        # XPU does not support graph currently.
         global global_graph_pool
-        if global_graph_pool is None:
+        if global_graph_pool is None and not current_platform.is_xpu():
```

Comment:
Shouldn't this be handled in `current_platform.graph_pool_handle`?

Reply:
thanks. move this to `XPUPlatform.get_graph_pool_handle()`